dogfooding/lib/screens/call_stats_screen.dart

```diff
@@ -0,0 +1,291 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_dogfooding/app/user_auth_controller.dart';
+import 'package:flutter_dogfooding/di/injector.dart';
+import 'package:flutter_dogfooding/screens/stats_latency_chart.dart';
+import 'package:flutter_dogfooding/theme/app_palette.dart';
+import 'package:stream_video_flutter/stream_video_flutter.dart';
+
+class CallStatsScreen extends StatefulWidget {
+  const CallStatsScreen({
+    super.key,
+    required this.call,
+  });
+
+  final Call call;
+
+  @override
+  State<CallStatsScreen> createState() => _CallStatsScreenState();
+}
+
+class _CallStatsScreenState extends State<CallStatsScreen> {
+  final _userAuthController = locator.get<UserAuthController>();
```

Comment:
Shouldn't we `dispose` this?

Reply:
it's a singleton so not here, it will be disposed when locator is cleaned up

Reply:
But if it is a singleton, we do not really have any state to track here, shouldn't we consider switching to `StatelessWidget`?

Reply:
you are right, fixed